java/src/IceGridGUI/src/main/java/com/zeroc/IceGridGUI/LiveDeployment/Node.java

```diff
@@ -269,16 +269,22 @@ protected String getDefaultFileName()
     Utils.ExpandedPropertySet expand(PropertySetDescriptor descriptor, String applicationName, Utils.Resolver resolver)
     {
         Utils.ExpandedPropertySet result = new Utils.ExpandedPropertySet();
-        result.references = new Utils.ExpandedPropertySet[descriptor.references.length];
+        java.util.ArrayList<Utils.ExpandedPropertySet> references = new java.util.ArrayList<Utils.ExpandedPropertySet>();
 
-        int i = 0;
         for(String ref : descriptor.references)
         {
-            result.references[i++] = expand(findNamedPropertySet(resolver.substitute(ref), applicationName),
-                                            applicationName, resolver);
+            PropertySetDescriptor resolved = findNamedPropertySet(resolver.substitute(ref), applicationName);
+            if (resolved == null)
+            {
+                getRoot().getCoordinator().getCommunicator().getLogger().warning(
+                    "Unable to resolve property set '" + ref + "' in application '" + applicationName + "'");
```

Comment:
I think we prefer `` `...' ``
```suggestion
                    "Unable to resolve property set `" + ref + "' in application `" + applicationName + "'");
```